homeassistant/components/shelly/coordinator.py

```diff
@@ -145,9 +145,10 @@ def async_setup(self, pending_platforms: list[Platform] | None = None) -> None:
             name=self.name,
             connections={(CONNECTION_NETWORK_MAC, self.mac)},
             manufacturer="Shelly",
-            model=MODEL_NAMES.get(self.model, self.model),
+            model=MODEL_NAMES.get(self.model),
```

Comment:
This dict lookup had a fallback before (`self.model`) which is now removed. Was that intended?

Reply:
No, that should not be removed, thanks for noticing

Reply:
Yes, this is intended. Currently we show `model_id` as `model` for devices which `model name` is not yet added in `aioshelly`. With this change `model` will be `None` for such devices.

Reply:
I first thought the create device will break, but checking this it is ok to pass `None` so this change is good